### PR TITLE
Reactivation of Travis CI tests

### PR DESCRIPTION
**Pull request by jcwinkler**
@sizmailov I reactivated the CI tests running on travis. The following topics are addressed:

- [x] Add of Inkscape 0.92.5 + TexText 0.11.1 snippets
- [x] Add Inkscape 1.0 + TexText 1.0 snippets
- [x] Add UTF tests
- [x] Add alignment tests
- [x] Tested with Inkscape 1.0
- [x] Tested on Linux Ubuntu 18.04 with Python 3.6
- [x] Tested on Linux Ubuntu 20.04 with Python 3.8
- [x] Tested on Windows 8.1 with Python 3.8.2
- [x] Tested on Travis-CI (my account, Ubuntu 18.04 with Python 3.6)
- [x] Tested on Travis-CI (Ubuntu 20.04 with Python 3.8)

### CODE REVIEW

**Comment by sizmailov** [comment]
Hi, I've looked at change, looks reasonable.

**Comment by jcwinkler** [comment]
Actually, test #3 for TexText 1.0 fails, see #245

File: .travis.yml [hunk 1]
```diff
@@ -14,33 +13,47 @@ matrix:
...
+      dist: focal
+      python: '3.8'
+      addons:
+        apt:
+          packages:
+          - python3-gi
+          - python3-gi-cairo
+          - gir1.2-gtk-3.0
+          - texlive-latex-base
+          - texlive-lang-cyrillic
```
Comment (sizmailov) [inline .travis.yml:39]: I don't think it would test anything _in textext_. The #240 is about environment configuration.

File: textext/base.py [hunk 2]
```diff
@@ -809,8 +821,12 @@ def import_group_color_style(self, src_svg_ele):
                                                 "fill-opacity"] and value.lower() != "none"}
 
             for it in self.iter():
+                color_style_update_dict = color_style_dict
+                # Ensure that simple strokes are also colored if the the group has a fill color
+                if "stroke" in it.style and "fill" in color_style_dict:
+                    color_style_update_dict["stroke"] = color_style_dict["fill"]
                 # Update style
-                it.style.update(color_style_dict)
+                it.style.update(color_style_update_dict)
```
Comment (sizmailov) [inline textext/base.py:829]: Needs to be rebased on top of #246

File: .travis.yml (hunk 1 above)
Comment (jcwinkler) [reply .travis.yml:39]: I think at least the "flow" of the characters is tested if the configuration is correct.

**Pull request merged**